.pre-commit-config.yaml

```diff
@@ -29,6 +29,7 @@ repos:
         args: 
           - "--autofix"
           - "--no-sort-keys"
+        exclude: ^tests/test_data/
```

Comment:
This exclusion just applies to the `pretty-format-json` hook, which we don't want to run on the test data because it would auto-fix/standardize all the indentation of the test JSON files.